CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v8.0.0]
+
+### Changed 
```

Comment:
Should we mention the users and data service?

Reply:
I don't think so, the user service is not changed regarding utf-8 support. And the data service should ony differ from the other node service with the `<` and the `>`.